mmv1/third_party/tgc_next/tfplan2cai/convert.go

```diff
@@ -0,0 +1,61 @@
+package tfplan2cai
+
+import (
+	"context"
+	"fmt"
+
+	"go.uber.org/zap"
+
+	"github.com/GoogleCloudPlatform/terraform-google-conversion/v6/pkg/caiasset"
+	"github.com/GoogleCloudPlatform/terraform-google-conversion/v6/pkg/tfplan2cai/resolvers"
+)
+
+// Options struct to avoid updating function signatures all along the pipe.
+type Options struct {
```

Comment:
`ConvertUnchanged` is removed from the struct, as it is not useful.

Reply:
Could you elaborate on this?

Reply:
Checked a few spots and don't see any dependencies on this.